src/net.cpp

```diff
@@ -1509,13 +1509,73 @@ void CConnman::QueryDNSSeeds()
     LogPrintf("%d addresses found from DNS seeds\n", found);
 }
 
+void CConnman::QueryFixedSeeds(std::chrono::microseconds start)
```

Comment:
8b1f718211b19fcf59b933c0cae7e237309913ad

Nothing is queried in this function, so why does it have this name?

Reply:
Ah sure it's done in the next commit. Maybe a commit message could mention it, or an in-code todo deleted in the next commit :)

Reply:
I will rename it to `ProcessFixedSeeds()` (I did it mostly so that it sounds similar to `QueryDNSSeeds`):
Even with the last commit, we don't actually query the fixed seeds in this function or thread, we just add them to `m_addr_fetches`, and ThreadOpenConnection will make the actual addrfetch connections that could be described as "querying".